src/makei/cvtsrcpf.py

```diff
@@ -36,6 +36,104 @@ def __init__(
 
         self.tolower = tolower
         self.ibmi_json_path = save_path / ".ibmi.json"
+        self.store_member_text = text
+
+    def check_keyword_in_file(self, file_path: str, keyword: str, line_limit: str) -> bool:
+        with open(file_path, 'r') as file:
+            for line_number, line in enumerate(file, start=1):
+                if keyword.lower() in line.lower():
+                    return True
+                if line_number == line_limit:
+                    break
+        return False
+    
+    # for free form rpg, write_on_line = 1
+    def insert_line(self, file_path, content, start_comment_characters: str, end_comment_characters: str, write_on_line: int, start_column: int, end_column: int) -> bool:
+        try:
+            if end_column <= start_column: return False
+            with open(file_path, 'r+') as file:
+                lines = file.readlines()
+                lines.insert(write_on_line, '\n')
+
+                starting_whitespace = 0 if start_column == 0 else start_column - 1
+                ending_whitespace = (end_column) - (starting_whitespace + len(start_comment_characters + content + end_comment_characters))
+
+                lines[write_on_line] = (' ' * starting_whitespace) + start_comment_characters + content + (' ' * ending_whitespace) + end_comment_characters + '\n'
+                file.seek(0)
+                file.writelines(lines)
+            return True
+        except:
+            return False
+        
+    def import_member_text(self, file_path: str, member_text: str, member_extension: str) -> bool:
+        text_comment_exists = self.check_keyword_in_file(file_path, 'TEXT', 15)
```

Comment:
I would at least look for '%TEXT', but that could be there for other reasons. 
Can we check that it is after a %METADATA